parser/parse_doc.m

```diff
@@ -0,0 +1,88 @@
+function docStruct = parse_doc(functionPath)
+%PARSE_DOC Generates a structure from the input function's documentation.
```

Comment:
Maybe it's something that was agreed previously, but is it necessary to have a struct? Isn't it a better idea to have a dictionary (map) so you can input any information needed in the header like examples, etc. and make a test at the end of the parse to check that minimal sections are present?

Reply:
I decided to go with a struct mainly because it was something I knew how to use. 😉 I'm not familiar with dictionaries but if you think that might be a better solution I'd be glad to discuss about it ! Is [this](https://www.mathworks.com/help/matlab/map-containers.html) what you are referring to ?

Reply:
Yes, that's exactly what I'm thinking about. I think maps or set depending on what we want could be more versatile.